components/scrapd-home-section/component.js

```diff
@@ -31,13 +36,17 @@ const ScrapdHomeSection = props => {
     padding: '1em'
   }));
 
-  const Text = styled.div({
-    maxWidth: '75%',
-    minWidth: '75%'
-  });
+  const Text = styled('div')`
+    ${mq({
+      maxWidth: ['99%', '80%'],
+      minWidth: ['99%', '80%'],
```

Comment:
I think you should leave the value at `75%` for the desktop mode.

```js
maxWidth: ['99%', '75%'],
minWidth: ['99%', '75%'],
```

That would preserve the images alternating left/right.

Reply:
Completed.